crates/rpc/src/v03/method/simulate_transaction.rs

```diff
@@ -20,6 +20,7 @@ use super::common::prepare_handle_and_block;
 #[derive(Deserialize, Debug)]
 pub struct SimulateTrasactionInput {
     block_id: BlockId,
+    #[serde(alias = "transactions")]
```

Comment:
Can we alias the other way around rather? Since `transactions` will become the correct value (and just makes more sense).

Reply:
And maybe add a comment on why the alias is there.